Sources/web3swift/Convenience/Base58.swift

```diff
@@ -49,41 +49,44 @@ struct Base58 {
             zerosToRemove += 1
         }
         base58.removeFirst(zerosToRemove)
-        
+
         while 0 < zerosCount {
             str = "\(str)1"
             zerosCount -= 1
         }
-        
+
         for b in base58 {
-            //str = "\(str)\(base58Alphabet[String.Index(encodedOffset: Int(b))])"
             str = "\(str)\(base58Alphabet[String.Index(utf16Offset: Int(b), in: base58Alphabet)])"
         }
-        
+
         return str
     }
-    
+
     // Decode
     static func bytesFromBase58(_ base58: String) -> [UInt8] {
         // remove leading and trailing whitespaces
         let string = base58.trimmingCharacters(in: CharacterSet.whitespaces)
-        
         guard !string.isEmpty else { return [] }
-        
-        var zerosCount = 0
-        var length = 0
+
+        // count leading "1"'s [decodes directly to zero bytes]
+        var leadingZeros = 0
```

Comment:
Comment says "count leading "1"'s" while the variable is named `leadingZeros` but it looks like it's actually counting leading ones. Probably the variable name is wrong.

Reply:
Leading binary bytes of 0 are encoded as ASCII 1's... It's confusing I know, that's why I added the context in the brackets "[decodes directly to 0's]". the variable name makes more sense, for when it gets used later in the code . (pretty much every reference bit of code I found that does the count and allocate seems to call the variable see variant of "zeros", it was originally "zerosCount" without any contextual comment)